src/components/approval-type/ApprovalType.vue

```diff
@@ -117,148 +114,147 @@ export default class ApprovalType extends Vue {
     courtNumRef: FormIF
   }
 
-  /** Draft court order number. */
+  /** The initial court order number. */
   @Prop({ default: '' }) readonly courtOrderNumber!: string
 
-  /** Whether approved by the registrar. */
+  /** Whether restoration was approved by the registrar. */
   @Prop({ default: false }) readonly approvedByRegistrar!: boolean
 
-  /** filing name used in radio options. */
-  @Prop({ default: 'restoration' }) readonly filingType!: string
+  /** Whether restoration was approved by court order. */
+  @Prop({ default: false }) readonly approvedByCourtOrder!: boolean
 
-  /** Show only the court order option; remove via registrar option. */
-  @Prop({ default: false }) readonly isCourtOrderOnly!: boolean
```

Comment:
I replaced this with `approvedByCourtOrder` (similar to approvedByRegistrar), which keeps the flag `isCourtOrderRadio`, below, functionally separate -- they do separate things instead of being dependent on each other.